pingouin/nonparametric.py

```diff
@@ -222,16 +225,20 @@ def mwu(x, y, alternative='two-sided', **kwargs):
         Association and the American Statistical Association, 25(2),
         101–132. https://doi.org/10.2307/1165329
 
+    .. [5] Campbell, M. J. & Gardner, M. J. (1988). Calculating confidence 
```

Comment:
Could you add in the "Notes" section a one line explanation of the CI method?

Reply:
Sure, I'll give that a go. Like I said, I'm not a statistician, so that'll have to be proof-read by someone